ovn-org/ovn-kubernetes
# Use ip mapped mac address for internal management port

#### What this PR does and why is it needed
Currently for management port of internal type, a randomly generated MAC address will be used, while for dpu case, the IP of management port will be mapped to mac address which in turn will be assigned to the port.
This commit changes the behavior of internal management port to make it align with that of dpu backed management port.

#### Which issue(s) this PR fixes
None

#### Special notes for reviewers
None

#### How to verify it
Check mac address of interface `ovn-k8s-mp0`, make sure it in the pattern of `0a:58:mm:xx:yy:zz`, where mm:xx:yy:zz is the hexadecimal form of management port's IP address.

#### Details to documentation updates
None

#### Description for the changelog
For management port of internal type, generate MAC address by mapping its IP address to hexadecimal form.

#### Does this PR introduce a user-facing change?
No
